addons/imjp94.yafsm/scenes/transition_editors/TransitionLine.gd

```diff
@@ -3,6 +3,8 @@ extends "res://addons/imjp94.yafsm/scenes/flowchart/FlowChartLine.gd"
 const Transition = preload("../../src/transitions/Transition.gd")
 const ValueCondition = preload("../../src/conditions/ValueCondition.gd")
 
+const hi_res_font: Font = preload("res://addons/imjp94.yafsm/assets/hi-res-font.tres")
```

Comment:
Path to font is not updated to `res://addons/imjp94.yafsm/assets/fonts/sans_serif.tres`